src/views/blocks/event-datetime.php

```diff
@@ -35,10 +35,14 @@
 );
 
 $time_format    = tribe_get_time_format();
-$date_format    = tribe_get_date_format( $use_yearless_format );
+$date_format    = tribe_get_date_format( true );
```

Comment:
It looks like a logic fault, couldn't we just do the following?

```suggestion
$date_format    = tribe_get_date_format( ! $use_yearless_format );
```

Reply:
Yeah that neater, I've updated this.